src/odo.ts

```diff
@@ -101,14 +101,14 @@ export class Command {
     static listCatalogServices () {
         return `odo catalog list services -o json`;
     }
-    static listStorageNames(project: string, app: string, component: string) {
+    static listStorageNames() {
         return `odo storage list -o json`;
     }
     static printOcVersion() {
         return 'oc version';
     }
-    static listServiceInstances(project: string, app: string) {
-        return `oc get ServiceInstance -o jsonpath="{range .items[?(.metadata.labels.app\\.kubernetes\\.io/part-of == \\"${app}\\")]}{.metadata.labels.app\\.kubernetes\\.io/instance}{\\"\\n\\"}{end}" --namespace ${project}`;
+    static listServiceInstances(context: string) {
+        return `odo service list -o json --context ${context}`;
```

Comment:
Restore previous parameters for the method and use them with --project and --app options. Do not forget to remove --context option.